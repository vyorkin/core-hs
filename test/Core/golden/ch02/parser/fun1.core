-- Functions 1

const x y = x
compose f g = \x. f (g x)
