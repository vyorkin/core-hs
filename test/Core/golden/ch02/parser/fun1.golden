[(Name "compose",[Name "g",Name "f"],ELam [Name "x"] (EAp (EVar (Name "f")) (EAp (EVar (Name "g")) (EVar (Name "x"))))),(Name "const",[Name "y",Name "x"],EVar (Name "x"))]